{"uuid": "1e28406d-39c8-4767-85a6-747593d707a9", "children": ["6e7a3683-ee4f-4859-9dad-6f8436139302"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "AttributeError: 'LocalConfig' object has no attribute 'DEVI_'\n", "trace": "  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\pc\\Desktop\\Challenge Gaston Ferreyra\\Mobile-test\\conftest.py\", line 39, in driver\n    opts.device_name = config.DEVI_\n                       ^^^^^^^^^^^^\n"}, "start": 1759458336097, "stop": 1759458336097}], "start": 1759458336097, "stop": 1759458336189}